index.rst

```diff
@@ -10,7 +10,240 @@ This is complemented by the so called "run branch", which is a common branch use
 
 This technote describes the details of the Observing Environment and how it is managed by our team.
 
-Add content here
-================
+Introduction
+============
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+The distributed nature of the Vera Rubin Observatory Control System makes it challenging to manage shared application/library software packages between users and the control system.
+In order to mitigate this issue we have created to so called "observing environment".
+This is a collection of packages that are central for the observatory operation and spans components configurations, SAL Script and other user facing packages.
+The idea is that users and control system will have access to a centralized common collection of packages that can be centrally managed and audited.
+
+Auditing capability is central to the observing environment as it allows us to identify what was being used by the control system at every single time, and also allows us to identify who made changes and when.
+
+In addition to the collection of packages and utilities to help manage the versions of the packages that comprises the observing environment, we have also instituted a procedure to allow multiple users to contribute to the system at a particular time.
+This is done by a common branch, centralized branch name that users can easily identify and contribute to.
+
+Here we descrive the procedures adopted by the run branch as well as how to manage their deployment.
+
+The Run Branch
+==============
+
+For every sprint (with a standard duration of 2 weeks), the observatory support team will create a ticket associated with the observatory operations support.
+The ticket number is used to define the associated run branch following the project guideline, i.e., ``tickets/<ticket-number>``.
+Branches are created in an as need base on the repositories that comprises the observing environment.
+It is not recommended to use the run branch for repositories outside the observing environment.
+However, in some cases (where hot patches are required), this might also be necessary.
+
+At the end of every sprint a member of the support team will curate the contributions from the run branche of each repository and either open pull requests to merge these contributions or work with the author of the contributions to get them merged.
+Nevertheless, not all contributions made to the run branch will be merged back during this process.
```

Comment:
I think you can strike Nevertheless here, it is implied in the previous sentence.